go/vt/vttest/vtprocess.go

```diff
@@ -142,11 +141,8 @@ func (vtp *VtProcess) WaitStart() (err error) {
 	vtp.proc.Args = append(vtp.proc.Args, vtp.ExtraArgs...)
 	vtp.proc.Env = append(vtp.proc.Env, os.Environ()...)
 	vtp.proc.Env = append(vtp.proc.Env, vtp.Env...)
-
-	if testing.Verbose() {
-		vtp.proc.Stderr = os.Stderr
-		vtp.proc.Stdout = os.Stdout
-	}
+	vtp.proc.Stderr = os.Stderr
+	vtp.proc.Stdout = os.Stdout
```

Comment:
as a middle ground, how would we feel about https://pkg.go.dev/testing#Testing

```suggestion
	if !testing.Testing() || testing.Verbose() {
		vtp.proc.Stderr = os.Stderr
		vtp.proc.Stdout = os.Stdout
	}
```

Reply:
Makes sense to me!

Reply:
Great improvement. Thanks @ajm188!